src/hmem_cuda.c

```diff
@@ -309,8 +310,8 @@ int cuda_hmem_init(void)
 	if (ret != FI_SUCCESS)
 		goto dl_cleanup;
 
-	ret = cuda_gdrcopy_hmem_init();
-	if (ret == FI_SUCCESS) {
+	gdrcopy_ret = cuda_gdrcopy_hmem_init();
+	if (gdrcopy_ret == FI_SUCCESS) {
```

Comment:
The else clause needs to be updated to reference gdrcopy_ret instead of ret as well.

Reply:
Thank you! Fixed in new revision.